grafana/common/PG_Overview.json

```diff
@@ -112,14 +112,14 @@
         },
         "textMode": "auto"
       },
-      "pluginVersion": "7.4.5",
+      "pluginVersion": "7.5.12",
       "repeat": "pgnodes",
       "repeatDirection": "h",
       "scopedVars": {
         "pgnodes": {
           "selected": false,
-          "text": "alpha_ip16_pg1",
-          "value": "alpha_ip16_pg1"
+          "text": "golf_ip16_pg1",
```

Comment:
I see three occurrences `grep '"\.5"' . -rn`
```
./common/PG_Overview.json:50:              "value": ".5"
./common/PG_Overview.json:169:              "value": ".5"
./common/PG_Overview.json:289:              "value": ".5"
```
May be just edit file and change it to 0.5 and eliminate other changes ?

Reply:
Should only be one place to actually change it I think. The other `".5"` values are the grafana version. Will try and do that instead.

Reply:
Kind of odd that the export only changed the value in one place, but there are actually 3 places the mapping is set in the json code. Fixed!